project_multifamily_beta/housing_characteristics/Water Heater.tsv

```diff
@@ -1,61 +1,61 @@
-Other Fuel	CR03	0.490000	0.000000	0.020000	0.030000	0.000000	0.000000	0.160000	0.000000	0.100000	0.040000	0.000000	0.020000	0.150000
-Other Fuel	CR04	0.820000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.110000	0.000000	0.000000	0.070000
-Other Fuel	CR05	0.670000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.050000	0.000000	0.230000	0.050000
-Other Fuel	CR06	0.900000	0.000000	0.000000	0.050000	0.000000	0.000000	0.000000	0.000000	0.000000	0.050000	0.000000	0.000000	0.000000
-Other Fuel	CR07	0.720000	0.000000	0.000000	0.060000	0.000000	0.000000	0.050000	0.000000	0.030000	0.060000	0.000000	0.000000	0.080000
-Other Fuel	CR08	0.870000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.100000	0.000000	0.000000	0.030000
-Other Fuel	CR09	0.930000	0.000000	0.020000	0.020000	0.000000	0.000000	0.000000	0.000000	0.000000	0.020000	0.000000	0.020000	0.000000
-Other Fuel	CR10	0.800000	0.000000	0.000000	0.110000	0.000000	0.000000	0.000000	0.000000	0.000000	0.090000	0.000000	0.000000	0.000000
-Other Fuel	CR11	0.320000	0.000000	0.000000	0.360000	0.000000	0.000000	0.000000	0.000000	0.000000	0.320000	0.000000	0.000000	0.000000
-Propane	CR02	0.560000	0.000000	0.010000	0.010000	0.000000	0.000000	0.000000	0.000000	0.000000	0.420000	0.000000	0.000000	0.000000
-Propane	CR03	0.120000	0.000000	0.000000	0.000000	0.000000	0.000000	0.090000	0.000000	0.000000	0.730000	0.000000	0.060000	0.000000
-Propane	CR04	0.480000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.500000	0.000000	0.020000	0.000000
-Propane	CR05	0.320000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.680000	0.000000	0.000000	0.000000
-Propane	CR06	0.600000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.400000	0.000000	0.000000	0.000000
-Propane	CR07	0.330000	0.000000	0.000000	0.030000	0.000000	0.000000	0.000000	0.000000	0.000000	0.650000	0.000000	0.000000	0.000000
-Propane	CR08	0.620000	0.000000	0.000000	0.000000	0.000000	0.000000	0.020000	0.000000	0.000000	0.340000	0.000000	0.030000	0.000000
-Propane	CR09	0.670000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.310000	0.000000	0.020000	0.000000
-Propane	CR10	0.300000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.700000	0.000000	0.000000	0.000000
-Propane	CR11	0.250000	0.000000	0.000000	0.030000	0.000000	0.000000	0.000000	0.000000	0.000000	0.690000	0.000000	0.000000	0.030000
+Dependency=Heating Fuel	Dependency=Location Region	Option=Electric Standard	Option=Electric Premium	Option=Electric Tankless	Option=Gas Standard	Option=Gas Premium	Option=Gas Tankless	Option=Oil Standard	Option=Oil Premium	Option=FIXME Oil Indirect	Option=Propane Standard	Option=Propane Premium	Option=Propane Tankless	Option=Other Fuel
+Electricity	CR02	0.780000	0.000000	0.000000	0.150000	0.000000	0.000000	0.000000	0.000000	0.000000	0.060000	0.000000	0.010000	0.000000
```

Comment:
@joseph-robertson Why does every line show up as changed? New lines? I thought Tony's script was specifically to address this.

Reply:
good question

Reply:
looks ok for singlefamilydetached Water Heater...